test/functional/cmdLineTests/criu/src/org/openj9/criu/DeadlockTest.java

```diff
@@ -90,7 +90,7 @@ public static void checkpointDeadlock() {
 		t1.start();
 
 		CRIUSupport criuSupport = new CRIUSupport(path);
-		criuSupport.registerPreSnapshotHook(() -> {
+		criuSupport.registerPreCheckpointHook(() -> {
 			synchronized (lock) {
 				System.out.println("Presnapshot hook inside monitor");
```

Comment:
I suggest changing this test message `Presnapshot` to `PreCheckpoint` as well, similarly for other occurrences of `PreSnapshotHook` and `PostSnapshotHook`.